python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/ethernet-interfaces.j2

```diff
@@ -329,11 +329,15 @@
 | Interface | Group Name | Direction |
 | --------- | ---------- | --------- |
 {%         for link_tracking_interface in link_tracking_interfaces %}
+{%             set link_tracking_group_name = [] %}
```

Comment:
I don't think you are using this variable anymore? Same under port-channel interfaces.

Reply:
Fixed!